app/domain/authentication/authn_gce/resource_restriction.rb

```diff
@@ -0,0 +1,17 @@
+module Authentication
+  module AuthnGce
+
+    class ResourceRestriction
```

Comment:
IMO this class is missing some comments like we have for OIDC and Azure. I think we can benefit with giving examples like we did with Azure
```
    # For example, if `resources` includes the AzureResource:
    #   - type: "subscription-id"
    #   - value: "some-subscription-id"
```

Reply:
just to calrify, you mean the comment should be in `validate_resource_restriction`, correct?

Reply:
I see we have 3 classes ResourceRestriction in the codebase, for authn-k8s,authn-azure, and for authn-gce. So I think it would be helpful to give examples of type and value like we do in the other authenticators

Reply:
we don't have a `ResourceRestriction` class for `authn-azure` and `authn-k8s`. in those modules the corresponding class is `<something>Resource`. the comment that you referred to above is from `validate_resource_restrictions`. would you want that comment to be also in `authn-gce/validate_resource_restrictions`?

Reply:
Authn-Azure:
```
module Authentication
  module AuthnAzure
     Class ResourceRestrictions
```
I thought was just like we wrote it in ResourceRestrictions, given an exaimple for what `type` and `value` would be

Reply:
yes, but we don't really have a corresponding class in gce. we have a planned effort to try and generalize this so we can handle it there